pkgs/servers/computing/slurm/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, pkgconfig, curl, python, munge, perl, pam, openssl,
-  ncurses, mysql, gtk }:
+  ncurses, mysql, gtk, lua, hwloc, numactl, extraDeps ? [] }:
```

Comment:
@adevress is it infeasible to encode optional dependencies explicitly as optional parameters? I think at least `extraDeps` needs a comment explaining what the expected value is, it's not a good interface currently, IMO.